plugins/out_stackdriver/stackdriver.c

```diff
@@ -729,7 +731,41 @@ static int get_stream(msgpack_object_map map)
     return STREAM_UNKNOWN;
 }
 
-static int pack_json_payload(int operation_extracted, int operation_extra_size,
+static void validate_insert_id(msgpack_object * subobj, const msgpack_object * o, 
```

Comment:
Can you just return the enum `insert_id_status`? this way the return value is simpler?

Also make the parameter name `subobj` and `o` more clear - e.g. `insert_id_value` and `obj` ?

Reply:
Done.